src/assg07-library.cpp

```diff
@@ -74,8 +91,22 @@ void swap(int& a, int& b)
   a = b;
   b = temp;
 }
-
-
+void swap(char& a, char& b)
```

Comment:
Missing function documentation here.  You were asked to copy/add the documentation for these functions in assignment 07.